src/blockrelay/graphene_set.h

```diff
@@ -156,6 +158,8 @@ class CGrapheneSet
             READWRITE(shorttxidk0);
             READWRITE(shorttxidk1);
         }
+        if (version >= 2)
+            READWRITE(ibltSalt);
```

Comment:
can I give you a salt that much later triggers the iblt.h line 142 exception (salt too big based on n_hashes)?

Reply:
With 4eee6a1 it should not be possible because any 32-bit salt value is valid for the IBLT.